index.html

```diff
@@ -32,9 +32,9 @@ <h5 class="ui_taglimainne">Get Your Own Design</h5>
     <div class="main_div">
         <div class="left_div">
             <div class="box1">
-                <p class="content">Get Started</p>
+                <span class="content">Get Started</span>
```

Comment:
Why are you using span here? `span` is an inline element

Reply:
yes. to cover the width of text